frontend/public/components/masthead-toolbar.jsx

```diff
@@ -597,7 +597,7 @@ class MastheadToolbarContents_ extends React.Component {
               <ToolbarItem className="visible-xs-block">
                 <NotificationBadge
                   aria-label="Notification Drawer"
-                  onClick={drawerToggle}
+                  onClick={this.drawerToggle}
```

Comment:
Is this right? It looks like it comes from props?

Reply:
@spadgett you are right

Reply:
@spadgett good catch, when i tested i didn't test mobile and didn't catch it.